test/unit/sql/index.test.js

```diff
@@ -232,6 +233,38 @@ describe(Support.getTestDialectTeaser('SQL'), () => {
         });
       });
     }
+
+    if (current.dialect.supports.index.include) {
+      it('include columns with unique index', () => {
+        expectsql(sql.addIndexQuery('User',  {
+          name: 'email_include_name',
+          fields: ['email'],
+          include: ['first_name', 'last_name'],
```

Comment:
could you add a test where `include` is a literal, and `include` is an array of literals? We try to support literal everywhere as an escape hatch

```ts
include: [literal('first_name')],

//

include: literal('(first_name, last_name)'),
```

it should add them as-is without further processing

Reply:
Done